Quake/gl_vidsdl.c

```diff
@@ -29,6 +29,9 @@ Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
 #include "resource.h"
 #include "SDL.h"
 #include "SDL_syswm.h"
+#ifdef VK_USE_PLATFORM_XCB_KHR
```

Comment:
If this is accepted, be sure to update the readme:

Debian: 	libx11-xcb-dev
Arch Linux: libx11